backend/app/api/v1/endpoints/ocp/ocpJobs.py

```diff
@@ -32,9 +32,10 @@ async def jobs(
         description="End date for searching jobs, format: 'YYYY-MM-DD'",
         examples=["2020-11-15"],
     ),
-    pretty: bool = Query(False, description="Output content in pretty format."),
+    pretty: bool = Query(False, description="Output contet in pretty format."),
     size: int = Query(None, description="Number of jobs to fetch"),
     offset: int = Query(None, description="Offset Number to fetch jobs from"),
+    sort: str = Query(None, description="To sort fields on specified direction"),
```

Comment:
The `sort` query parameter doesn't seem to be used here??

Reply:
While merging, it got lost